answers.yaml.example

```diff
@@ -0,0 +1,7 @@
+dotfiles_repo: git@github.com:myuser/dotfiles.git
+fedora_packages:
+  my-fave-text-editor
+  my-alternate-web-browser
+fedora_packages_to_remove:
+  something-i-dislike
+  that-other-thing-i-dislike
```

Comment:
I now have the main playbook looking for answers.yaml then answers.yml and doing include_vars, and if you pass ```-e "answers_url=https://raw.githubusercontent.com/j2sol/ansibox/master/ansibox-answers.yaml'``` your answerfile would get downloaded and populated :)